x-pack/plugins/security/public/lib/privilege_utils.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { KibanaPrivilegeSpec } from '../../common/model';
+
+/**
+ * Determines if the passed privilege spec defines global privileges.
+ * @param privilegeSpec
+ */
+export function isGlobalPrivilegeDefinition(privilegeSpec: KibanaPrivilegeSpec): boolean {
```

Comment:
How would you feel about moving these to the models, so that we can more easily ask questions like this?